src/main/java/dev/cammiescorner/arcanuscontinuum/Arcanus.java

```diff
@@ -145,8 +145,10 @@ public void onInitialize(ModContainer mod) {
 		});
 
 		ResourceLoaderEvents.END_DATA_PACK_RELOAD.register(ctx -> {
-			if(ctx.server() != null)
-				Arcanus.refreshSupporterData(ctx.server(), true);
+			try (MinecraftServer server = ctx.server()) {
```

Comment:
wrapping this in a try-with-resources statement will cause java to close the server object at the end of that statement

Reply:
Oops